bin/init

```diff
@@ -2,7 +2,7 @@
 
 # This script will setup the database and then start the rails application
 
-set -e
+set -euo pipefail
```

Comment:
Chances are, something in this script is expected to fail - and we'll need top update more code to make it work with the stricter environment.